scripts/download-core.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 set -o pipefail
 
-: ${REALM_CORE_VERSION:=0.96.2} # set to "current" to always use the current build
+: ${REALM_CORE_VERSION:=0.97.0} # set to "current" to always use the current build
```

Comment:
Thanks for the info. We should update object store with the newer version as well (that's where I pulled this from).